.github/workflows/run-api-analytics-tests.yml

```diff
@@ -0,0 +1,79 @@
+name: Run analytics api tests
+env:
+  # This is to make sure Maven don't timeout fetching dependencies. See: https://github.com/actions/virtual-environments/issues/1499
+  MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=125
+
+on:
+  pull_request:
+    types: [ opened, labeled, synchronize ]
+  schedule:
+    - cron: "16 1 * * *" # Run at 01:16 AM
+concurrency:
+  group: ${{ github.workflow}}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  api-test:
+    env:
+      CORE_IMAGE_NAME: "dhis2/core-dev:local"
+      PR_NUMBER: ${{ github.event.number }}
+      DOCKER_CHANNEL: "dhis2/core-pr"
+      SIERRA_LEONE_DB_PATH: "/tmp/db/sierra-leone"
+      SIERRA_LEONE_DB_VERSION: "2.39.0"
```

Comment:
Should we use the [dev](https://databases.dhis2.org/sierra-leone/dev/dhis2-db-sierra-leone.sql.gz) version here?

Reply:
@maikelarabori since you chose this version, I think you know :smi

Reply:
@radnov, this was chosen as a baseline DB. The analytics tests run on top of the DB and are very sensitive to data and dates. So, we need to keep the same DB version to get the same expected results for every execution.
If we choose "dev", we will have a different DB version (along with different data and dates) for every release. In this case, the results are very likely to change each time we cut a new version of the "dev" DB. Consequently, the test's expectations are likely to fail.
Taking a specific DB version will ensure we keep the same DB version so tests remain consistent with the data.